gammapy/data/tests/test_observation_summary.py

```diff
@@ -0,0 +1,52 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from astropy.coordinates import SkyCoord
+
+from ...data import DataStore, ObservationTable, ObservationTableSummary
+
+from ...utils.testing import data_manager, requires_data, requires_dependency
+
+def init_summary():
+    """Init summary table with gammapy-extra data
+
+    Returns
+    -------
+    summary = `gammapy.data.ObservationTableSummary`
+        Summary table 
+    """
+    
+    data_store = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
+    observations = data_store.obs_table
+    summary = ObservationTableSummary(observations,SkyCoord.from_name('crab'))
+    return summary
+
+@requires_data('gammapy-extra')
+def test_str():
+    """Test if str is well computed"""
```

Comment:
I would suggest you remove all these docstrings on your test functions.
They don't add extra info wrt the code.